src/representation_theory/Action.lean

```diff
@@ -140,6 +140,14 @@ def mk_iso {M N : Action V G} (f : M.V ≅ N.V) (comm : ∀ g : G, M.ρ g ≫ f.
   { hom := f.inv,
     comm' := λ g, by { have w := comm g =≫ f.inv, simp at w, simp [w], }, }}
 
+@[priority 100]
+instance is_iso_of_hom_is_iso {M N : Action V G} (f : M ⟶ N) [is_iso f.hom] : is_iso f :=
+by { convert is_iso.of_iso (mk_iso (as_iso f.hom) f.comm), ext, refl, }
```

Comment:
The only question I have is whether this instance is safe -- are there any loops this can cause? I don't know whether it's possible it might apply in this situation, but I noticed https://leanprover-community.github.io/mathlib_docs/category_theory/monoidal/Mon_.html#Mon_.quiver.hom.hom.category_theory.is_iso

Otherwise,
bors d

Reply:
I think it's safe. That instance causes an even-less-likely-to-succeed typeclass search, so I don't think it can loop.